blocks/header-nav-chain-block/_index.scss

```diff
@@ -1,6 +1,16 @@
 @use "@wpmedia/arc-themes-components/scss";
 
 .b-header-nav-chain {
+	&--scrolled {
+		@include scss.block-components("header-nav-chain-scrolled");
+		@include scss.block-properties("header-nav-chain-scrolled");
+	}
+
+	&--not-scrolled {
```

Comment:
looks like this `not-scrolled` is not being used. if so, it can be removed, right?